.github/workflows/test-namespace-darwin.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
     name: Bazel Test macOS Apple Silicon
     timeout-minutes: 120
     runs-on: namespace-profile-darwin # profile created in namespace console
+    if: ${{ github.repository == 'dfinity/ic' }} # only run on public repo, not private
```

Comment:
```suggestion
    if: ${{ github.repository == 'dfinity/ic' }} # only run on public repo, not private since Namespace runners are not configured there, so these CI jobs get stuck otherwise.
```